tests/foreman/sys/test_pulp3_filesystem.py

```diff
@@ -6,9 +6,9 @@
 
 :CaseLevel: System
 
-:CaseComponent: Pulp
+:CaseComponent: Repositories
```

Comment:
I wonder if this module should rather stay under Pulp as these tests relates to the back-end service. :thinking: 
@shweta83

Reply:
I agree. This module is more Pulp related.